src/test/resources/MakeAppointment.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Make Appointment
+
+  An a user
+  I want to sign in
+  So I can use the app
+  After that I fill fields to make my appointment
+
+  @automated
+  Scenario Outline: User can make appointment with valid value
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
+    Then user sees appointment page
+    When user select facility as "<facilityBox>"
+    And user "<action>" apply
+    And user chooses "<program>"
+    And user decides "<date>"
+    And user adds "<comment>"
```

Comment:
chỗ này c nên viết rõ ra là `And user chooses program as "<program>"`, như thế thì ai đọc cũng hiểu, những cái dưới tương tự